southwest_checkin.py

```diff
@@ -35,12 +45,18 @@
 wait = WebDriverWait(driver,10)
 
 # go directly to the checkin page
-driver.get("https://www.southwest.com/flight/retrieveCheckinDoc.html")
+driver.get("https://mobile.southwest.com/check-in")
 
 # get text boxes for check in
-confirmation_num = driver.find_element_by_id("confirmationNumber")
-first_name = driver.find_element_by_id("passengerFirstName")
-last_name = driver.find_element_by_id("passengerLastName")
+while True:
+	try:
+		confirmation_num = driver.find_element_by_name("recordLocator")
+		first_name = driver.find_element_by_name("firstName")
+		last_name = driver.find_element_by_name("lastName")
+		break;
+	except NoSuchElementException:
+		print("loading checkin...")
+		sleep(0.1)
```

Comment:
Sometimes, on the mobile website, the page will be marked as "loaded" before it's rendered the input fields; this'll catch that and wait for them to show up.